src/components/materialWrappers/Select.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Dropdown } from 'react-native-material-dropdown'
```

Comment:
Screenshot of this component? We probably have some other UI that can be used instead.

Reply:
1/10 discussion - Leave in for now, Paul will research.